tests/gradients/test_parameter_shift.py

```diff
@@ -287,7 +287,8 @@ def test_with_gradient_recipes(self):
         assert tapes[3].get_parameters(trainable_only=False) == [1.0, 2.0, 2 * 3.0 + 2, 4.0]
         assert tapes[4].get_parameters(trainable_only=False) == [1.0, 2.0, 3 * 3.0 + 3, 4.0]
 
-    def test_recycled_unshifted_tape(self):
+    @pytest.mark.parametrize("ops_with_custom_recipe", [[0], [1], [0, 1]])
+    def test_recycled_unshifted_tape(self, ops_with_custom_recipe):

```

Comment:
This test is generalized here. For the cases `[0]` and `[1]`, it fails on master (with the extension using `fn(qml.execute(...))` below)